exercises/pythagorean-triplet/src/Triplet.hs

```diff
@@ -1,7 +1,10 @@
 module Triplet (isPythagorean, mkTriplet, pythagoreanTriplets) where
 
-isPythagorean = error "You need to implement this function."
+isPythagorean :: (Int, Int, Int) -> Bool
```

Comment:
if this is done, then the reasoning in https://github.com/exercism/xhaskell/blob/master/exercises/pythagorean-triplet/.meta/DONT-TEST-STUB for why the tests would fail to build is false, so if we do this please delete that file so that the stub gets tested.

If we choose the type of `mkTriplet` as `(Int, Int, Int)` rather than letting the student choose, do we even need `mkTriplet`? Shouldn't we just delete `mkTriplet` then? I think the only point of it is to give the student the choice.

Why did we give this choice, anyway?